tests/queries/0_stateless/02415_all_new_functions_must_be_documented.reference

```diff
@@ -562,6 +562,7 @@ positionCaseInsensitive
 positionCaseInsensitiveUTF8
 positionUTF8
 pow
+printf
```

Comment:
WTF

Reply:
I confused it with `aspell-dict.txt` and didn't notice that there's no embedded documentation